CHANGELOG.md

```diff
@@ -76,7 +76,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - [#485](https://github.com/nf-core/sarek/pull/485) - `--skip_qc`, `--skip_markduplicates` and `--skip_bqsr` is now `--skip_tools`
 - [#538](https://github.com/nf-core/sarek/pull/538) - `--sequencing_center` is now `--seq_center`
-- [#538](https://github.com/nf-core/sarek/pull/538) - `--markdup_java_options` has been removed
+- [#539](https://github.com/nf-core/sarek/pull/539) - `--annotate_tools` has been removed
+- [#539](https://github.com/nf-core/sarek/pull/539) - `--cadd_cache`, `--cadd_indels`, `--cadd_indels_tbi`, `--cadd_wg_snvs`, `--cadd_wg_snvs_tbi` have been removed
+- [#539](https://github.com/nf-core/sarek/pull/539) - `--genesplicer` has been removed
```

Comment:
ok so the way to select this now is by setting `--tools vep` and then all the files needed?

Reply:
yeah, will need some more docs about it, but all files that we supply for VEP go into this vep_extra_files channel.
Need to check out if it behave properly and don't get consume whenever